project/src/main/java/com/google/sps/servlets/PostEventServlet.java

```diff
@@ -56,7 +56,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     SimpleDateFormat formatter = new SimpleDateFormat("MMM dd, yyyy");
     long dateTimestamp = 0;
     try {
-      Date formattedDate = formatter.parse(date);
+      Date formattedDate = formatter.parse(date.substring(4));
```

Comment:
Why you taking part of the date only?

Reply:
the first three chars are for the day of the week which we needed to readability purposes for the date but would cause issues when being converted

Reply:
could you just change formatter? If you have only days in the input you could use new SimpleDateFormat("dd");

Reply:
I'm not sure what you mean. We only need the remaining part of the date so the mm dd, yyyy. the substring is just removing the ddd that I added to the pop up calendar thing because we dont need the ddd for the date element

Reply:
Wait, that makes sense. Done.